docs/conf.py

```diff
@@ -13,23 +13,34 @@
 import os
 from datetime import datetime
 import sys
-sys.path.insert(0, os.path.abspath('../ush'))
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                os.pardir)))
 sys.path.append(os.path.abspath("./_ext"))
 print(sys.path)
 
+from metplus import __version__
 
 # -- Project information -----------------------------------------------------
 
 project = 'METplus'
 
 author = 'UCAR/NCAR, NOAA, CSU/CIRA, and CU/CIRES'
 
-# the stable version, displayed on front page of PDF
-version = 'develop'
-verinfo = version
-
 # The full version, including alpha/beta/rc tags
-release = f'{version}'
+# i.e. 4.0-beta1-dev
+release = __version__
+
+# the stable version, displayed on front page of PDF extract X.Y version
+# from release by splitting the string into a list
+# using - as the delimeter, then getting the 1st item of the list
+# if version is beta, rc, and/or dev then set version to develop for
+# the documentation built for develop (not release)
+if len(release.split('-')) > 1:
+    version = 'develop'
+else:
+    version = f"{release.split('-')[0]}"
+
+verinfo = version
 
 release_year = '2020'
```

Comment:
Would be nice if there were a way to avoid having to manually update an explicit date. Any ideas?

Reply:
You can get the current date easily with Python, but if we had to push updates to an older version, the date would be overwritten. We could add an argument 'release' to the script that is used when creating a release that will get the current date and save it into a text file in the docs directory that will be used if it exists on an future update. This would have to be done after the main_vX.Y branch is created so it is only added to that branch. Also, we would have to add instructions when creating a release to commit that file. Thoughts?

Reply:
Yeah, good points you raise. If manual intervention is required anyway, might as well just keep it as is.